service/worker/migration/activities.go

```diff
@@ -897,13 +897,13 @@ func (a *activities) WaitCatchup(ctx context.Context, params CatchUpParams) erro
 		ActiveCluster: descResp.ReplicationConfig.GetActiveClusterName(),
 	}
 
-	activeAckIDOnShard, err := a.getActiveClusterReplicationStatus(ctx, waitCatchupRequest)
+	activeAckIDOnShard, err := a.getActiveClusterReplicationStatus(ctx, waitCatchupRequest) // from B, A ack level
```

Comment:
what do those comments mean?

Reply:
will remove. This is note from another dicussion.